### PR TITLE
gnparser 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by dimus**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gnparser.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  homepage "https://github.com/gnames/gnparser"
+  url "https://github.com/gnames/gnparser/archive/v1.0.1.tar.gz"
+  sha256 "92f532c60cc325806c5c8bfd08590cf2905305a88f3d23eae395575fd281b67b"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    man1.install "man/gnparser.1"
+    system "go", "build", "-o", "#{bin}/gnparser", "./gnparser"
```
Comment (carlocab) [inline Formula/gnparser.rb:12]:
```suggestion
    system "go", "build", *std_go_args, "./gnparser"
```

**Review by carlocab — CHANGES_REQUESTED** [review]
Thanks, @dimus. One minor suggestion.

**Comment by carlocab** [comment]
Thanks again for your PR, @dimus. This seems like a nice piece of software, but it does not meet the requirements for inclusion in homebrew/core. The CI log reads:
```
gnparser:
  * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
Error: 1 problem in 1 formula detected
```
You may submit this formula again when this requirement is met.

If you wish, you can also host this formula in your own tap. Doing this will be easy, now that you have a formula to add to it. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

**Comment by dimus** [comment]
@carlocab, to meet this requirement, does a repo need to have all 3 contidions? > 30 forks, >30 watchers, >75 stars, or just one of them?

This is a biological package, not for computer scientists, and most of users are not on even on GitHub.

**Comment by dimus** [comment]
hm I see

**Comment by carlocab** [comment]
I believe you only need to meet only one of the conditions, but you can look at the audit code in brew to double check.

**Pull request closed without merging**